OWML.Launcher/SocketListener.cs

```diff
@@ -70,15 +69,32 @@ private void ListenToSocket(ref TcpListener server)
 
         private void PrintOutput(byte[] bytes, int count)
         {
-            var data = Encoding.ASCII.GetString(bytes, 0, count);
-            var objects = data.Split(new string[] { ";;" }, StringSplitOptions.None);
+            var json = Encoding.ASCII.GetString(bytes, 0, count);
+
+            var data = JsonConvert.DeserializeObject<SocketMessage>(json);
 
-            if (objects[1] == Constants.QuitKeyPhrase)
+            if (data.Message == Constants.QuitKeyPhrase)
```

Comment:
I guess this magic phrase could be a message type instead...

Reply:
done!